.ko.yaml

```diff
@@ -0,0 +1,2 @@
+baseImageOverrides:
+  github.com/tektoncd/pipeline/cmd/resolvers: cgr.dev/chainguard/git@sha256:566235a8ef752f37d285042ee05fc37dbb04293e50f116a231984080fb835693
```

Comment:
this would make all resolvers image having to use the git image... isnt that a bit much for example the http-resolver would def not need it...

Reply:
If I understand correctly all of the resolvers run as plugins together in the same pod(s). So at least while the some resolvers will have unnecessary access to the `git` binary, all of the resolver pods will need access to the binary if the git resolver is enabled. Not sure if that's _that_ much better though